demos/client-bundle-example/src/jsBundleTest.html

```diff
@@ -15,7 +15,7 @@
             <div
                 class="procaptcha"
                 data-theme="light"
-                data-sitekey="5HUBceb4Du6dvMA9BiwN5VzUrzUsX9Zp7z7nSR2cC1TCv5jg"
+                data-sitekey="5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw"
```

Comment:
I don't think this needs to change

Reply:
Yes, this is my mistake
Sorry

Reply:
I checked this again but I didn't change this

Reply:
Will have changed as part of a build step then - fine for it to change

Reply:
Sorry guys, this actually needs to remain as `5HUBceb4Du6dvMA9BiwN5VzUrzUsX9Zp7z7nSR2cC1TCv5jg`. The reason is the tests rely on this *live* site key 

https://github.com/prosopo/captcha/blob/a4c47396fa38d01661695d3a55d8b0a06db776c9/.github/workflows/publish.yml#L423

`5HUBceb4Du6dvMA9BiwN5VzUrzUsX9Zp7z7nSR2cC1TCv5jg` is registered as a dapp/sitekey in our live rococo contract.

Unfortunately, the dev setup script (`npm run setup`) searches for these site key values and replaces them with the development site key. We could fix this by doing the search replace during the CICD [here](https://github.com/prosopo/captcha/blob/a4c47396fa38d01661695d3a55d8b0a06db776c9/.github/workflows/publish.yml#L423) and replacing the value with a github secret.

Reply:
For now, @Vengi717, please revert this change.